samples/scenarios/src/main/java/quarks/samples/scenarios/iotf/range/sensor/IotfRangeSensor.java

```diff
@@ -0,0 +1,192 @@
+package quarks.samples.scenarios.iotf.range.sensor;
+
+import static quarks.analytics.math3.stat.Statistic.MAX;
+import static quarks.analytics.math3.stat.Statistic.MEAN;
+import static quarks.analytics.math3.stat.Statistic.MIN;
+import static quarks.analytics.math3.stat.Statistic.STDDEV;
+
+import java.io.File;
+import java.util.Date;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.pi4j.io.gpio.Pin;
+import com.pi4j.io.gpio.RaspiPin;
+
+import quarks.analytics.math3.json.JsonAnalytics;
+import quarks.connectors.iot.IotDevice;
+import quarks.connectors.iot.QoS;
+import quarks.connectors.iotf.IotfDevice;
+import quarks.function.Supplier;
+import quarks.providers.direct.DirectProvider;
+import quarks.providers.direct.DirectTopology;
+import quarks.topology.TStream;
+import quarks.topology.TWindow;
+
+public class IotfRangeSensor {
```

Comment:
This is a new file so can the indenting be made consistent within it wrt use of spaces vs tabs?  Things aren't lining up.  Note, as recently noted in DEVELOPMENT.md, the desire is no hard tabs.